lms/envs/devstack.py

```diff
@@ -385,6 +385,7 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 ############## Settings for Microfrontends  #########################
 LEARNING_MICROFRONTEND_URL = 'http://localhost:2000'
 ACCOUNT_MICROFRONTEND_URL = 'http://localhost:1997'
+PROFILE_MICROFRONTEND_URL = 'http://localhost:1995'
```

Comment:
2U specific -- I think we want to add this change to https://github.com/edx/devstack/blob/master/py_configuration_files/lms.py

Reply:
I will make that PR, thanks. I knew there was a reason I asked you to be the reviewer 😆